che-theia-java-extension/scripts/get-extension.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2018 Red Hat, Inc.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which is available at http://www.eclipse.org/legal/epl-2.0.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+const fs = require('fs');
+const process = require('child_process');
+const path = require('path');
+
+const packagePath = path.join(__dirname, '..');
+const version = '0.0.1-SNAPSHOT';
+const archiveExtensionCoreName = `jdt.ls.extension.core-${version}.jar`;
+const archiveExtensionAPIName = `jdt.ls.extension.api-${version}.jar`;
+const downloadDir = 'download';
+const downloadPath = path.join(packagePath, downloadDir);
+const archiveExtensionCorePath = path.join(downloadPath, archiveExtensionCoreName);
+const archiveExtensionAPIPath = path.join(downloadPath, archiveExtensionAPIName);
+
+function getExtension() {
+    return new Promise((resolve, reject) => {
+        const command = 'mvn dependency:copy ';
```

Comment:
Why are we not doing this as a build-time operation? Our two bundles together come to around 200k, no need to add an extra post-install step that might go wrong.

Reply:
Also, we might do without maven altogether for non-dev scenarios like I did in the main Java bundle (download release for normal post-install, but use maven to get latest snapshot).

Reply:
Moved to `build` lifecycle. 
We can do it without maven, especially for non-dev scenario when we'll make release for che-ls-jdt, because for now we don't have location of these two bundles.